hdate/date.py

```diff
@@ -322,3 +401,10 @@ def get_omer_string(omer):
                 omer_string += u'שני ימים '
     omer_string += u'לעומר'
     return omer_string
+
+
+def hdate_from_hebrew_date(hebrew_date):
```

Comment:
Instead of doing this, simply change the __init__ to accept either Hebrew date or Gregorian date.